### DISCUSSION TITLE
Is there an Early Stopping option?

### DISCUSSION THREAD

**Discussion by jackkwok** [open-mmlab/mmsegmentation#2394] [General]
I want to do early stop the training if the validation loss doesn't decrease for N iterations. Is early stopping implemented?  If not, I will implement it.

**Comment by sanath-tech** [comment]
Even i am struck with the same problem , but i wanted to early stop when validation loss starts increasing for some N iterations

**Comment by xiexinch** [comment]
Hi @jackkwok,
Sorry for the late reply, MMSeg 1.x will support this feature in the future.

https://github.com/open-mmlab/mmengine/pull/739
